cms/static/sass/partials/_variables.scss

```diff
@@ -39,6 +39,11 @@ $f-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
 // ====================
 $transparent: rgba(0,0,0,0); // used when color value is needed for UI width/transitions but element is transparent
 
+// +Colors - UXPL new pattern library colors
+// ====================
+$primary-accent: rgb(14, 166, 236);
```

Comment:
Good question-- I don't know! @clrux what should "primary-accent" be now? I was trying to define variables using the pattern library names to make it easier to understand how it links up with the pattern library.

I actually don't need primary-dark anymore and can remove it.